src/Mobs/Enderman.cpp

```diff
@@ -5,9 +5,60 @@
 #include "Enderman.h"
 #include "../Entities/Player.h"
 #include "../LineBlockTracer.h"
+#include "../BlockInfo.h"
 
+bool FindTeleportDestination(cWorld *a_World, Vector3i a_Centre, const int a_HalfCubeWidth, const int a_HeightRequired, const unsigned int a_NumTries, Vector3i &a_Destination)
```

Comment:
This is a great Idea. I think this could be a static function of cEntity or cPawn

Reply:
Yep that seems more logical, thanks!